packages/runtime/src/ctx.ts

```diff
@@ -18,12 +22,13 @@ import {
 	ZERO_ADDRESS_STR,
 } from "./lib/constants";
 import { pyExt, tealExt } from "./lib/pycompile-op";
-import { calculateFeeCredit } from "./lib/txn";
+import { calculateFeeCredit, isEncTxApplicationCreate, isEncTxAssetConfig, isEncTxAssetCreate, isEncTxAssetDeletion, isEncTxAssetFreeze, isEncTxAssetOptIn, isEncTxAssetReconfigure, isEncTxAssetRevoke } from "./lib/txn";
```

Comment:
How about create method name `getTransactionType(txn)`. Take al sdk Txn object and return the `wtypes.TransactionType`? 

Example:
```ts
let txnType = getTransactionType(txn);  // txnType = wtypes.TransactionType.DeployApp
```